apps/condo/domains/organization/components/EmployeeProfileForm/index.tsx

```diff
@@ -108,6 +132,29 @@ export const EmployeeProfileForm = () => {
                                     >
                                         <Input/>
                                     </Form.Item>
+                                    <Form.Item noStyle dependencies={['role']}>
+                                        {
+                                            ({ getFieldValue })=> {
+                                                const role = getFieldValue('role')
+                                                const selectedRole = find(employeeRoles, { id: role })
+
+                                                if (selectedRole.name === TechnicianRoleName)
+                                                    return (<Col span={24}>
+                                                        <Form.Item
+                                                            name={'specializations'}
+                                                            label={SpecializationsLabel}
+                                                            labelAlign={'left'}
```

Comment:
You can use labelAlign on a form instead of writing it to every Form.Item

Reply:
This is not referring to my changes, I will open new pr with refactoring of all Form.Item's